docs/solana/rewardable-entities.mdx

```diff
@@ -0,0 +1,419 @@
+---
+id: rewardable-entities
+hide_title: true
+sidebar_label: Rewardable Entities (Hotspots, Mappers, and more)
+slug: /solana/rewardable-entites
+hide_table_of_contents: true
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+:::warning 🚧 This document is a work in progress. 🚧
+
+The documentation on Solana and the upcoming Migration is under development and subject to change.
+
+:::
+
+# Rewardable Entities (Hotspots, Mappers, and more)
+
+Following the transition to Solana, all actors in the network that receive tokens are considered "Rewardable Entities."
+
+While, as of writing this document, hotspots are the only rewardable entity, this will change with the addition of mappers and other rewardable categories.
```

Comment:
Will staking and HSTs not be implemented at launch as rewardable entities? If not how will they be implemented?

Reply:
Fanout wallet, using Hydra. See metaplex program library